packages/map-template/src/components/MapTemplate/MapTemplate.scss

```diff
@@ -30,6 +34,10 @@
         .venue-selector__button {
             display: none;
         }
+
+        .switch {
```

Comment:
The "switch" class name is a bit too generic, I think. Consider renaming it to something more specific.

Reply:
✅

Reply:
👏